vuepress/docs/next/tutorials/create/ms/add-access-controls.md

```diff
@@ -1,109 +1,161 @@
----
-sidebarDepth: 2
----
 # Role Based Access Controls
 
 ## Overview
-This tutorial guides you through adding access controls to your existing Entando project. Security experts recommend following a practice known as `Defense in Depth` where security controls are placed in each layer of an architecture. This tutorial will help you setup such controls in both the frontend and backend of your Entando application. 
+Experts recommend following a practice known as Defense in Depth where security controls are placed in each layer of an architecture. This tutorial guides you through adding access controls to your existing Entando project, in both the frontend and backend of your Entando Application. 
 
-For the purpose of this tutorial we'll use the simple Conference application from [this tutorial](./generate-microservices-and-micro-frontends.md) as a starting point. Please work through that tutorial if you have not already. 
+The simple Conference application found in the [Generate Microservices and Micro Frontends tutorial](./generate-microservices-and-micro-frontends.md) is used as a starting point. We recommend working through that tutorial for background and context. 
 
-The basic security setup for a blueprint-generated application allows any authenticated user to access the functionality contained in the MFEs and/or microservices. Our business requirement for this tutorial is to define two kinds of users in our application - `Conference Users` who can view the Conferences in the tableWidget, and `Conference Admins` who can view and also delete Conferences from the tableWidget. 
+The basic security setup for a blueprint-generated application allows any authenticated user to access the functionality contained in the MFEs and/or microservices. This tutorial defines two user roles for our application:
 
-## Tutorial
-Let's start by securing the list of Conferences so only our two user roles can view the list.
+- `conference-user`: Permitted to view the Conferences in the tableWidget
+- `conference-admin`: Permitted to view Conferences in the tableWidget, and also to delete Conferences from the tableWidget 
 
-1. Edit `ConferenceResource.java` located in the `src/main/java/com/<ORG>/<NAME>.web.rest` directory. Modify the REST API `Conference:getAllConferences` method by adding the following annotation. 
+
+## Apply and Verify Access Controls
+
+### Step 1: Secure the Conference list
+
+The list of Conferences must be visible to only the `conference-user` and `conference-admin` user roles. 
+
+1. Go to the `src/main/java/com/YOUR-ORG/YOUR-NAME.web.rest` directory
+2. Open `ConferenceResource.java` 
+3. Modify the REST API `Conference:getAllConferences` method by adding the following annotation:
 ```
     @PreAuthorize("hasAnyAuthority('conference-user','conference-admin')")
     public List<Conference> getAllConferences() {
```

Comment:
Does this line match your starting code? I thought it was a little different, maybe getAllConferences(Pageable pageable)?

Reply:
it doesn't; i actually removed the second line and rephrased because that tripped me up, like i was supposed to add the second line somewhere, too, when really it's used here to show where @PreAuthorize belongs